src/codegen/utilities/function_helpers.py

```diff
@@ -11,12 +11,37 @@
     "CIGPS": "CI_GPS",
 }
 
+EXCLUDED_FUNCTIONS = [
+    "AddNetworkDevice",
+    "DeleteNetworkDevice",
+    "ReserveNetworkDevice",
+    "UnreserveNetworkDevice",
+    "AddCDAQSyncConnection",
+    "AreConfiguredCDAQSyncPortsDisconnected",
+    "AutoConfigureCDAQSyncConnections",
+    "GetAnalogPowerUpStates",
+    "GetAnalogPowerUpStatesWithOutputType",
+    "GetAutoConfiguredCDAQSyncConnections",
+    "GetDigitalLogicFamilyPowerUpState",
+    "GetDigitalPowerUpStates",
+    "GetDigitalPullUpPullDownStates",
+    "GetDisconnectedCDAQSyncPorts",
+    "RemoveCDAQSyncConnection",
+    "SetAnalogPowerUpStates",
+    "SetAnalogPowerUpStatesWithOutputType",
+    "SetDigitalLogicFamilyPowerUpState",
+    "SetDigitalPowerUpStates",
+    "SetDigitalPullUpPullDownStates",
+]
+
 
 def get_functions(metadata, class_name=""):
     """Converts the scrapigen metadata into a list of functions."""
     all_functions = deepcopy(metadata["functions"])
     functions_metadata = []
     for function_name, function_data in all_functions.items():
+        if "pythod_codegen_method" in function_data or function_name in EXCLUDED_FUNCTIONS:
```

Comment:
Is `pythod` a typo? Also I think it should be `"python_codegen_method" not in function_data` just going by the name.